### PR TITLE
Added short instructions and image on how to attach display

### PR DESCRIPTION
**Pull request by JamesH65**

### CODE REVIEW

File: hardware/display/README.md [hunk 1]
```diff
@@ -10,6 +10,12 @@ The DSI display is designed to work with all models of Raspberry Pi, however ear
 
 [Legacy support for Raspberry Pi 1 Model A/B](legacy.md)
 
+## Physical Installation 
+
+The following image shows how to attach the Raspberry Pi to the back of the Touch Display (if required), and how to connect both the data (Ribbon cable) and power (Red/Black wires) from the Raspberry Pi to the display. If you are not attaching the Raspberry Pi to the back of the display, take extra care when attaching the ribbon cable to ensure it is the correct way round. The black and red power lines should be attached to the GND and 5v pins respectively.
```
Comment (lurch) [inline hardware/display/README.md]: Probably don't need Capital Letters on Ribbon, Red and Black? :wink: 

Also: power lines -> power wires ?

File: hardware/display/README.md [hunk 2]
```diff
@@ -10,6 +10,12 @@ The DSI display is designed to work with all models of Raspberry Pi, however ear
...
+
+![DSI Display Connections](GPIO_power-500x333.jpg "DSI Display Connections")
```
Comment (lurch) [inline hardware/display/README.md]: Seems you forgot to add the image to this PR?

**Comment by JamesH65** [comment]
Everyone one with this? @lurch @6by9

**Comment by lurch** [comment]
The 'legacy' page mentions "Power the setup via the PWR IN micro-USB connector on the display board.". Should this page be saying the same thing? Or for the B+ onwards, does it not matter if the display powers the Pi, or the Pi powers the display?

**Comment by JamesH65** [comment]
@lurch Just spoken to a man who knows, and we would need to do some EMC testing to know which is better! That's scheduled, so best to wait until that is done before recommending one or the other. I'd be inclined to merge this PR (once everyone happy) and create an issue to update if necessary after the testing.

**Pull request merged**